core/src/test/scala/scalacss/full/InlineTest.scala

```diff
@@ -153,7 +153,28 @@ object MyInline3 extends StyleSheet.Inline {
     innerObject.andAgain.depth2)
 }
 
-  object InlineTest extends utest.TestSuite {
+object MyInlineWithKeyframes extends StyleSheet.Inline {
+  import dsl._
+
+  val s = style(
+    height(100 px),
+    width(30 px)
+  )
+
+  val kf1 = keyframes(
+    (0 %%) -> s,
+    (20 %%) -> kstyle(
+      height(150 px),
+      width(30 px)
+    ),
+    (100 %%) -> kstyle(
+      height(200 px),
+      width(60 px)
+    )
+  )
+}
```

Comment:
bip bop! This test is missing the most vital part: usage. Could you please test creating an `animation-name: blah` entry. There should also be a type-safe guarantee that when you create a keyframes group, that you can use it - meaning the user shouldn't have to rely on the same name-as-a-string in more than one place, and it should be immune to the keyframes group name being auto-renamed.